include/bluetooth/bluetooth.h

```diff
@@ -1213,7 +1228,110 @@ int bt_le_per_adv_sync_recv_enable(struct bt_le_per_adv_sync *per_adv_sync);
+/** Periodic Advertising options */
+enum {
+	/** Convenience value when no options are specified. */
+	BT_LE_PAST_OPT_NONE = 0,
+
+	/** Sync with Angle of Arrival (AoA) constant tone extension */
+	BT_LE_PAST_OPT_DONT_SYNC_AOA = BIT(0),
+
+	/** Sync with Angle of Departure (AoD) 1 us constant tone extension */
+	BT_LE_PAST_OPT_DONT_SYNC_AOD_1US = BIT(1),
+
+	/** Sync with Angle of Departure (AoD) 2 us constant tone extension */
+	BT_LE_PAST_OPT_DONT_SYNC_AOD_2US = BIT(2),
+
+	/** Do not sync to packets without a constant tone extension */
+	BT_LE_PAST_OPT_SYNC_ONLY_CONST_TONE_EXT = BIT(3),
+};
+
+struct bt_le_past_param {
+	/**
+	 * @brief Maximum event skip
+	 *
+	 * The number of periodic advertising packets that can be skipped
+	 * after a successful receive.
+	 */
+	uint16_t skip;
+
+	/**
+	 * @brief Synchronization timeout (N * 10 ms)
+	 *
+	 * Synchronization timeout for the periodic advertising sync.
+	 * Range 0x000A to 0x4000 (100 ms to 163840 ms)
+	 */
+	uint16_t timeout;
+
+	/** Periodic Advertising Sync Transfer options */
+	uint32_t options;
+};
+
+/**
+ * @brief Transfer the periodic advertising sync information to a peer device.
+ *
+ * This will allow another device to quickly synchronize to the same periodic
+ * advertising train that this device is currently synced to.
+ *
+ * @param per_adv_sync  The periodic advertising sync to transfer.
+ * @param conn          The peer device that will receive the sync information.
+ * @param service_data  Application service data provided to the remote host.
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+int bt_le_per_adv_sync_transfer(const struct bt_le_per_adv_sync *per_adv_sync,
+				const struct bt_conn *conn,
+				uint16_t service_data);
+
+
+/**
+ * @brief Transfer the information about a periodic advertising set.
+ *
+ * This will allow another device to quickly synchronize to periodic
+ * advertising set from this device.
+ *
+ * @param adv           The periodic advertising set to transfer info of.
+ * @param conn          The peer device that will receive the information.
+ * @param service_data  Application service data provided to the remote host.
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+int bt_le_per_adv_set_info_transfer(const struct bt_le_ext_adv *adv,
+				    const struct bt_conn *conn,
+				    uint16_t service_data);
+
+/**
+ * @brief Subscribe to periodic advertising sync transfers (PASTs).
+ *
+ * Sets the parameters and allow other devices to transfer periodic advertising
+ * syncs.
+ *
+ * @param conn    The connection to set the parameters for. If NULL default
+ *                parameters for all connections will be set. Parameters set
+ *                for specific connection will always have precedence.
+ * @param param   The periodic advertising sync transfer parameters.
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+int be_le_past_subscribe(const struct bt_conn *conn,
```

Comment:
The use of "past" is not well aligned, since we have per_adv and per_adv_sync it is strange to suddenly start using PAST. Also a mix is used now.

bt_le_per_adv_sync_transfer
bt_le_per_adv_sync_transfer_set_info
bt_le_per_adv_sync_transfer_subscribe
bt_le_per_adv_sync_transfer_unsubscribe

Maybe transfer could by shortened, `tx` or `xfer`

Reply:
also "be_le" should be "bt_le"

Reply:
I somewhat agree. 

I actually wanted to use `past` all around to keep the length down, but `bt_le_past` just seemed too generic, so on the transmitter side I used the full name and the abbreviation on the receiver. 

`tx` probably wouldn't work either, as `bt_le_per_adv_sync_tx` sounds more like a transmit function. `xfer` could be a decent alternative. 

I've updated all functions to be the full name, but kept the `PAST` abbreviation for enums and macros.

Reply:
I would have preferred the PAST_OPTs to also have the full name.

Using PAST in hci.h is fine, since this is an API for HCI driver writers so it is more of an export API compared to the rest of the headers.

Reply:
Updated :)

Reply:
Great, It says TRANSFOR though :)

Reply:
Good catch :D